packages/patternfly-4/react-charts/src/components/ChartTheme/themes/theme-donut.js

```diff
@@ -0,0 +1,24 @@
+// TODO Replace label props with PF css when available
+export const styles = {
+  label: {
+    position: 'absolute',
+    height: '100%',
+    subTitle: {
+      fill: '#bbb'
+    },
+    title: {
+      fontSize: 20
+    },
+    width: '100%'
+  }
+};
+
+// Donut properties to extend Victory theme
+export default {
+  pie: {
+    height: 230,
+    innerRadius: 98,
```

Comment:
Since we are calculating an innerRadius in the component, this shouldn't be necessary.

Reply:
I was keeping this as a default, in case height and width are overridden.

Reply:
If height/width are overridden, innerRadius should still be calculated from those values, right? If the calculated innerRadius doesn't meet the need of the user, they can override that too. I don't think there is any need for a static innerRadius value.

Reply:
The innerRadius will also be calculated if height and width are overridden. I'm thinking of an edge case where the value could be negative or the given height and width are zero. I just wanted to have a default.

Reply:
I've pushed a change to enure the innerRadius doesn't go negative.